modules/sdk-coin-xlm/test/unit/utils.ts

```diff
@@ -91,8 +91,46 @@ describe('Stellar Utils', () => {
       const decodedPrv = Utils.decodePrivateKey(userKeychain.prv).toString('hex');
       const decodedBackupPrv = Utils.decodePrivateKey(backupKeychain.prv).toString('hex');
 
-      decodedPrv.should.equal(rootKeychain.prv);
-      decodedBackupPrv.should.equal(backupRootKeychain.prv);
+      decodedPrv.should.equal(rootKeychain.prv.slice(0, 64));
+      decodedBackupPrv.should.equal(backupRootKeychain.prv.slice(0, 64));
+    });
+  });
+
+  describe('create Stellar keypairs', () => {
```

Comment:
Can you add non-happy path tests as well? What errors do we expect for invalid input, what if input is empty, etc.

Reply:
Added these tests into this PR. Also refactored these methods to have better error handling